EOSatelliteImagery/schema.json

```diff
@@ -0,0 +1,104 @@
+{

```

Comment:
Everything here  is right but there are some properties in the examples (example.json) that are not in the schema
'layer_categorization', 'layer_name', 'outputLocalServerPath', 'pilot', 'productFormat', 'productID', 'productType', 'productURL', 'relativeOrbit', 'sensorMode', 'values_explanation', 'waterBodyLocalId'

Reply:
Addtionally, I have removed some required attributes that you don't have in he examples. Otherwise is not going to validate.

Reply:
Thanks for the comments. 
- 'layer_categorization', 'layer_name' and 'values_explanation' are all contained into the 'contentInformation' attribute. Should I define them in a different way in the schema?
- The rest of the attributes should be ok now both in the example files and in schema.